plugins/inx/server_milestones.go

```diff
@@ -2,25 +2,71 @@ package inx
 
 import (
 	"context"
+	"time"
+
+	"github.com/pkg/errors"
 
 	"google.golang.org/grpc/codes"
 	"google.golang.org/grpc/status"
 
+	"github.com/gohornet/hornet/pkg/common"
+	"github.com/gohornet/hornet/pkg/dag"
 	"github.com/gohornet/hornet/pkg/model/milestone"
 	"github.com/gohornet/hornet/pkg/model/storage"
+	"github.com/gohornet/hornet/pkg/utils"
+	"github.com/gohornet/hornet/pkg/whiteflag"
 	"github.com/iotaledger/hive.go/events"
+	"github.com/iotaledger/hive.go/serializer/v2"
 	"github.com/iotaledger/hive.go/workerpool"
 	inx "github.com/iotaledger/inx/go"
+	iotago "github.com/iotaledger/iota.go/v3"
+)
+
+var (
+	ComputeWhiteFlagTimeout = 2 * time.Second
```

Comment:
Lets reuse the same config parameter like in the `debug` plugin.
Since we should move that debug API call to the rest API anyway, lets also move the config setting for that to a more generic config part.